aten/src/ATen/native/mkldnn/MKLDNNCommon.cpp

```diff
@@ -75,6 +75,19 @@ ideep::tensor itensor_view_from_dense(const Tensor& tensor) {
            ideep::tensor::data_type::f32},
           tensor.template data_ptr<float>()};
 }
+
+// Note in case the aten Tensor is a dense tensor, the retured ideep
+// tensor is just a view of the storage of the aten dense tensor, so
+// caller needs to make sure the aten dense tensor's lifetime is
+// longer than the ideep tensor.
+ideep::tensor get_mkldnn_tensor(const at::Tensor& tensor) {
+  if (tensor.is_mkldnn()) {
+    return at::native::itensor_from_mkldnn(tensor);
+  } else {
```

Comment:
What if tensor is sparse? We need to error out.

Reply:
**itensor_view_from_dense** will check whether the tensor is dense, if not, will error out.